src/window.rs

```diff
@@ -288,6 +288,13 @@ impl Window {
     pub fn set_title(&self, title: &str) {
         self.glutin_window.set_title(title);
     }
+
+    /// Set cursor visible
+    #[inline]
+    pub fn set_cursor_visible(&self, show: bool) {
+        self.glutin_window.set_cursor(if show { glutin::MouseCursor::Default }
```

Comment:
This should track the current visibility state to avoid redundant calls into glutin (which may be expensive).